assets/js/modules/analytics/datastore/settings.test.js

```diff
@@ -969,7 +974,7 @@ describe( 'modules/analytics settings', () => {
 				).toBe( false );
 			} );
 
-			it( 'should return false when the analytics-4 module is not active', () => {
+			it( 'should return false when the analytics-4 module is not connected', () => {
```

Comment:
This looks like the exact same test as the one below: https://github.com/google/site-kit-wp/pull/6787/files#diff-2313bc49b1ca0a5ad8ff075262dc398ef7a01adaad25ee3116132532d385b7feR1035

It'd be better to keep this test's spec name and test/verify what the expected outcome is when GA4 module is _connected, but not active_.

Reply:
Yes, removed 👍